AlexSuth/ncs-i502-capstone
# Add pages/routes

**What it does:**
Creates pages/routes for Home, About, Create Account, Edit Products, Login, Reset Password.

**To Test:**
1. Navigate to i502portal file
2. npm install and npm start
3. Browser should open to 'Home' - you can navigate to the other pages by changing the address in the browser (e.g. localhost:3000/about)
